### PR TITLE
delete some files

### PR DESCRIPTION
**Pull request by lnybrave**

### CODE REVIEW

**Review by lnysky — COMMENTED** [review]
is ok

File: utils/calc_time.py [hunk 1]
```diff
@@ -1,143 +0,0 @@
...
-        strBeginDay = "%04d%02d%02d0000" % (objDatetime.year, objDatetime.month, objDatetime.day)
-        strEndDay = "%04d%02d%02d2359" % (objDatetime.year, objDatetime.month, objDatetime.day)
-        lstRetTime.append([strBeginDay, strEndDay])
-        objDatetime = (objDatetime + objTimedelta)
-
-    lstRetTime[-1][1] = strEndTime
-    return lstRetTime
-
-#减少天数，例如：传入 2015-08-06 23:00:00和1 返回 2015-08-05 23:00:00
-def subtractDays(strTime, iDays):
```
Comment (liunianyon) [inline utils/calc_time.py]: can leave this fun

Comment (lnybrave) [inline utils/calc_time.py]: i do not think so

Comment (liunianyon) [inline utils/calc_time.py]: delete it

**Pull request merged**